src/hub.html

```diff
@@ -881,7 +881,7 @@
                     textColor: #fff;
                     backgroundColor: #2f80ed;"
                 position="0 0 0.01">
-                <a-entity text="value:Lift your headset to continue; width:1; align:center;" position="0 0 0.01"></a-entity>
+                <a-entity text="value:View your desktop to continue; width:1; align:center;" position="0 0 0.01"></a-entity>
```

Comment:
Suggestion: "Continue in your browser tab"

Reply:
I think I like this copy better for now, super hard to know what connects without user testing ofc. We should figure out a way to see people go through this flow